Source/Mosa.Compiler.Common/BinaryWriterExtensions.cs

```diff
@@ -46,11 +46,6 @@ public static void Write(this BinaryWriter writer, bool value, uint nativeSize)
 		Write(writer, BitConverter.GetBytes(value), nativeSize);
 	}
 
-	public static void Write(this BinaryWriter writer, sbyte value, uint nativeSize)
```

Comment:
Why was this removed?

Reply:
The function was not used and either way it was redundant because it was creating a new array, since it was just one byte.